### PR TITLE
Run swagger markdown generation in a profile.

### PR DESCRIPTION
**Pull request by hgschmie**
Only build the online documentation when rebuilding. Rebuilding the files in Docs is triggered
by using "-Pbuild-swagger-documentation"

### CODE REVIEW

**Comment by hgschmie** [comment]
rebased

**Comment by hgschmie** [comment]
brought in all the changes from #419 and actually make the endpoint links and the API sorting work. :-)

File: pom.xml [hunk 1]
```diff
@@ -5,7 +5,7 @@
   <parent>
     <groupId>org.basepom</groupId>
     <artifactId>basepom-standard-oss</artifactId>
-    <version>8</version>
```
Comment (hgschmie) [inline pom.xml]: this is required because it otherwise hits duplicate-finder-plugin issue 9.

**Comment by tpetr** [comment]
thanks!

**Pull request merged**